src/Microsoft.Health.Fhir.Core/Features/Conformance/CapabilityStatementBuilder.cs

```diff
@@ -79,6 +86,19 @@ public ICapabilityStatementBuilder UpdateRestResourceComponent(string resourceTy
                         Reference = $"http://hl7.org/fhir/StructureDefinition/{resourceType}",
                     },
                 };
+                if (!_modelInfoProvider.Version.Equals(FhirSpecification.Stu3))
```

Comment:
Is there a specific reason we are excluding Stu3 version?

Reply:
Thanks, I look properly this time, and STU3 has different way of displaying supporting profiles which I'd missed before.

Reply:
Not anymore!

---
In reply to: [605744661](https://github.com/microsoft/fhir-server/pull/1795#discussion_r605744661) [](ancestors = 605744661)